### PR TITLE
Adding widget translation model

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
Issue #: https://apps.itsm.gov.bc.ca/jira/browse/DESENG-514

*Description of changes:*
    - Added Widget translation model.
    - Added Widget translation API.
    - Added Unit tests.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-api/src/met_api/services/widget_translation_service.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            if not language_record:
+                raise ValueError('Language to translate was not found')
+
+            if pre_populate:
+                # prepopulate translation with base language data
+                WidgetTranslationService._get_default_language_values(widget, translation_data)
+
+            created_widget_translation = WidgetTranslationModel.create_widget_translation(translation_data)
+            return WidgetTranslationSchema().dump(created_widget_translation)
+        except IntegrityError as e:
```
Comment (Baelx) [inline met-api/src/met_api/services/widget_translation_service.py:55]: Will this be able to catch when we have two translations for the same widget? The widget_id and language_id combination needs to be unique

File: CHANGELOG.MD [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## March 06, 2024
+- **Task**Multi-language - Create simple widget translation tables & API routes [DESENG-514](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-514)
+    - Added Widget translation model.
+    - Added Widget translation API.
+    - Added Unit tests.
+
 ## March 04, 2024
```
Comment (Baelx) [inline CHANGELOG.MD:7]: Could I ask you to please combine the two Changelog entries for March 4th? I see that it was duplicated

**Review by Baelx — CHANGES_REQUESTED** [review]
Looks great! I just want to ensure that translations will have a unique widget_id + language_id combination.

Also could you please combine the two March 4 entries in the Changelog?

File: met-api/src/met_api/services/widget_translation_service.py (hunk 1 above)
Comment (VineetBala-AOT) [reply met-api/src/met_api/services/widget_translation_service.py:55]: Yes I tried to post two requests with the same widget_id and language_id, the second time it gives a BusinessException

**Review by Baelx — APPROVED** [review]

**Pull request merged**